requirements-dev.txt

```diff
@@ -1 +1,2 @@
 pylint
+pytest
```

Comment:
Just wondering whether we should capture versions as well?

I don't have a strong opinion either ways, we actually don't have versions in setup.py also.

Reply:
Currently, I assume the added benefit is quite low, because I do not think pytest is going to change their API in major ways. And if things are really about to break, that's what the CI build is for.